hsm/README.md

```diff
@@ -0,0 +1,109 @@
+Using HSM on SAGE proto
+=======================
+
+1. Connect to a client supporting multi-pool (as of today, client-[24-28]).
+
+2. Set the following environment variables:
+   
+   ```bash
+   export CLIENT_PROF_OPT="<0x7000000000000001:0x480>"   # profile id
+   export CLIENT_HA_ADDR="172.18.1.24@o2ib:12345:34:101" # ha-agent address
+   export CLIENT_LADDR="172.18.1.24@o2ib:12345:41:322"   # local address
+   export CLIENT_PROC_FID="<0x7200000000000001:0xdf>"    # process id
+   ```
+
+   Profile id of the cluster and ha-agent address on your client node can
+   be checked with `hctl mero status` command. As well as all addresses and
+   processes ids configured in the cluster. Consult with the cluster system
+   administrator about which of them you can use.
+
+3. Initialize the composite layout index:
+    
+   ```bash
+   $ c0composite "$CLIENT_LADDR" "$CLIENT_HA_ADDR" "$CLIENT_PROF_OPT" "$CLIENT_PROC_FID"
+   ```
+
+   Note: this should be done one time only after the cluster bootstrap.
+
+4. Configure pools ids of the tiers in ~/.hsm/config file:
+
+   ```
+   M0_POOL_TIER1 = <0x6f00000000000001:0xc74> # NVME
+   M0_POOL_TIER2 = <0x6f00000000000001:0xc8a> # SSDs
+   M0_POOL_TIER3 = <0x6f00000000000001:0xca5> # HDDs
+   ```
+
+   The exact ids can be taken from the output of `hctl mero status` command.
```

Comment:
Is multiple pool feature currently supported by Hare ?

Reply:
Just recently Valery added support for this to Hare, see https://github.com/Seagate/cortx-hare/pull/1373.

Reply:
@andriytk @madhavemuri The multipools patch hasn't been merged yet.  The branch is slowly rotting away... 🍂